content/guides/microservices/develop-rest-api-nodejs-mongodb-containers.md

```diff
@@ -0,0 +1,238 @@
+  curl -H "Content-Type: application/json" -X POST -d '{"name":"milk","quantity":"10"}' http://localhost:1337/item
+  ```
+  
+  You should see output similar to that shown below:
+
+  ![POST request](/images/guides/microservices/develop-rest-api-nodejs-mongodb-containers/api-post.png)
+
+* Check if the item record was created with a GET request:
+
+  ```bash
+  curl http://localhost:1337/item
+  ```
+
+  You should see output similar to that shown below:
+  
+  ![GET request](/images/guides/microservices/develop-rest-api-nodejs-mongodb-containers/api-get.png)
+
+  You can also connect to the running MongoDB container and use the *mongo* CLI to see the data in the MongoDB database.
+  
+  ```bash
+  docker exec -it mongodb /bin/bash
+  mongo --authenticationDatabase mydb -u myapp -p myapp mydb --eval "db.item.find()"
+  ```
+
+  You should see output similar to that shown below:
+
+  ![Database check](/images/guides/microservices/develop-rest-api-nodejs-mongodb-containers/db-get.png)
+
+* Modify the item record with a PUT request. Replace the ID placeholder in the command below with the document's unique identifier from the previous commands.
+
+  ```bash
+  curl -H "Content-Type: application/json" -X PUT -d '{"name":"milk","quantity":"5"}' http://localhost:1337/item/ID
+  ```
+  
+  You should see output similar to that shown below:
+  
+  ![PUT request](/images/guides/microservices/develop-rest-api-nodejs-mongodb-containers/api-put.png)
+
+* Delete the item record with a DELETE request:
+
+  ```bash
+  curl -H "Content-Type: application/json" -X DELETE http://localhost:1337/item/ID
+  ```
+
+  You should see output similar to that shown below:
+  
+  ![DELETE request](/images/guides/microservices/develop-rest-api-nodejs-mongodb-containers/api-delete.png)
+
+  You can also connect to the running MongoDB container and use the *mongo* CLI to confirm that the data has been deleted from the MongoDB database:
+
+  ```bash
+  docker exec -it mongodb /bin/bash
+  mongo --authenticationDatabase mydb -u myapp -p myapp mydb --eval "db.item.count()"
+  ```
+  
+  You should see output similar to that shown below:
+
+  ![Database check](/images/guides/microservices/develop-rest-api-nodejs-mongodb-containers/db-delete.png)
+  
+At this point, you have a working Node.js REST API integrated with a MongoDB database. However, it's currently only available on your local machine and only while the containers are running. This is fine for local development, but becomes a bottleneck when you want to make it available to others for review and testing. 
+
+That's where the [second part of this series](https://docs.bitnami.com/tutorials/deploy-rest-api-nodejs-mongodb-charts/) comes in. Continue adding features to your API and once you're happy with it, proceed to the concluding article and learn how to make your REST API available to a larger group by containerizing it, deploying it on Kubernetes and connecting it to a MongoDB service also running on Kubernetes. 
```

Comment:
Likewise, I assume this should link to the dev center version of the guide instead as well